### PR TITLE
Add visualize histogram

### PR DESCRIPTION
**Pull request by Manonedde**
Script to visualize the histogram of metric values (FA and MD) from a binary mask.
Data to test the script: https://drive.google.com/file/d/158eKOLS0IBn1gKjRHBn-lQhWfbwwAPAF/view?usp=sharing

### CODE REVIEW

**Comment by frheault** [comment]
Can I have a screenshot of the typical use case?

**Comment by Manonedde** [comment]
Here is a histogram of FA in WM : 
https://drive.google.com/file/d/1aYBy7XdchYBVx0KHdpAkUILYQZJn-JCz/view?usp=sharing

File: scripts/scil_visualize_histogram.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    p.add_argument('in_mask',
+                   help='Binary mask data to extract value.')
+    p.add_argument('n_bins', type=int,
+                   help='Number of bins to use for the histogram.')
+    p.add_argument('out_png',
+                   help='Output filename for the figure.')
+
+    hist = p.add_argument_group(title='Histogram options')
+    hist.add_argument('--title', default='Histogram',
+                      help='Use the provided info to histogram title.'
```
Comment (gabe1496) [inline scripts/scil_visualize_histogram.py:38]: Preferably, "[...] info _for the_ histogram title."

File: scripts/scil_visualize_histogram.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                           ' [%(default)s]')
+    hist.add_argument('--x_label',  default='x',
+                      help='Use the provided info to name x axis.')
+    hist.add_argument('--colors', default='#0504aa',
+                      help='Use the provided a color for bar plot.'
```
Comment (gabe1496) [inline scripts/scil_visualize_histogram.py:43]: Suggestion : "Use the provided info for the bars color."

**Review by gabe1496 — APPROVED** [review]

File: scripts/scil_visualize_histogram.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.in_metric, args.in_mask)
```
Comment (arnaudbore) [inline scripts/scil_visualize_histogram.py:58]: args.in_mask is also mandatory

Comment (arnaudbore) [reply scripts/scil_visualize_histogram.py:58]: check also output exist

**Review by mdesco — APPROVED** [review]
Looks good to me! Good job.

**Comment by arnaudbore** [comment]
Please remove what you wrote in `scripts/tests/__init__.py`.
Then I'll merge.

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**